core/src/main/resources/hudson/model/View/delete_lt.properties

```diff
@@ -1,2 +1,4 @@
-Are\ you\ sure\ about\ deleting\ the\ view?=Ar tikrai norite i\u0161trinti rodin\u012f?
-Yes=Taip
+# /jenkins-core/src/main/resources/hudson/model/View/delete_lt.properties

```

Comment:
If you are going to maintain this translation, I'm fine if your IDE injects something like that